tests/pages/firefox/accounts.py

```diff
@@ -2,24 +2,35 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
+from selenium.common.exceptions import TimeoutException
 from selenium.webdriver.common.by import By
+from selenium.webdriver.support import expected_conditions as expected
+from selenium.webdriver.support.ui import WebDriverWait as Wait
 
 from pages.firefox.base import FirefoxBasePage
-from pages.regions.download_button import DownloadButton
 
 
 class FirefoxAccountsPage(FirefoxBasePage):
 
     URL_TEMPLATE = '/{locale}/firefox/accounts/'
 
-    _download_button_locator = (By.ID, 'features-hero-download')
+    _download_buttons_locator = (By.CLASS_NAME, 'features-download')
     _create_account_form_locator = (By.ID, 'fxa-email-form')
 
     @property
-    def download_button(self):
-        el = self.find_element(*self._download_button_locator)
-        return DownloadButton(self, root=el)
+    def is_create_account_form_displayed(self):
+        wait = Wait(self, timeout=3)
```

Comment:
We shouldn't need to add any implicit waits like this, as our tests only run once the page has fully loaded and JS has executed.